src/plugin/regions/region.js

```diff
@@ -36,7 +36,7 @@ export class Region {
         // no styling or can be assigned an object containing CSS properties.
         this.handleStyle = params.handleStyle || {
             left: {},
-            right: {}
+            right: {},
```

Comment:
this extra comma is not needed.

Reply:
For now I have removed all the dangling commas that I had put.

Reply:
thanks, I enabled the `comma-dangle` eslint rule in master branch.